AutoCollection/Performance.ts

```diff
@@ -135,10 +141,29 @@ class AutoCollectPerformance {
                 totalIrq += irq;
             }
 
+            // Calculate % of total cpu time (user + system) this App Process used (Only supported by node v6.1.0+)
+            let appCpuPercent: number = undefined;
+            if (typeof (process as any).cpuUsage === 'function') {
+                const appCpuUsage = (process as any).cpuUsage();
+                const hrtime = process.hrtime();
+
+                const totalApp = ((appCpuUsage.user - this._lastAppCpuUsage.user) + (appCpuUsage.system - this._lastAppCpuUsage.system)) || 0;
+
+                if (typeof this._lastHrtime !== 'undefined' && this._lastHrtime.length === 2) {
```

Comment:
do we still need the typeof x !== 'undefined" trick for legacy browsers, or we are good to use !== undefined?

Reply:
Browsers aren't a concern for this SDK, so checking just against undefined should be fine. Though IMO just `!this._lastHrtime` is sufficient if we wanted to be really concise.